tests/unit/Mage/Adminhtml/Block/TemplateTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * OpenMage
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available at https://opensource.org/license/osl-3-0-php
+ *
+ * @category   OpenMage
+ * @package    OpenMage_Tests
+ * @copyright  Copyright (c) 2024 The OpenMage Contributors (https://www.openmage.org)
+ * @license    https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace OpenMage\Tests\Unit\Mage\Adminhtml\Block;
+
+use Generator;
+use Mage;
+use Mage_Adminhtml_Block_Template;
+use PHPUnit\Framework\TestCase;
+
+class TemplateTest extends TestCase
+{
+    public Mage_Adminhtml_Block_Template $subject;
+
+    public function setUp(): void
+    {
+        Mage::app();
+        // phpcs:ignore Ecg.Classes.ObjectInstantiation.DirectInstantiation
+        $this->subject = new Mage_Adminhtml_Block_Template();
+    }
+
+    /**
+     * @see Mage_Core_Model_Session::getFormKey()
+     * @group Mage_Adminhtml
+     * @group Mage_Adminhtml_Block
+     * @group runInSeparateProcess
+     * @runInSeparateProcess
+     */
+    public function testGetFormKey(): void
+    {
+        $this->assertIsString($this->subject->getFormKey());
+    }
+
+    /**
+     * @covers Mage_Adminhtml_Block_Template::isOutputEnabled()
+     * @dataProvider provideIsOutputEnabled
+     * @group Mage_Adminhtml
+     * @group Mage_Adminhtml_Block
+     */
+    public function testIsOutputEnabled(bool $expectedResult, string $moduleName): void
+    {
+        $this->assertSame($expectedResult, $this->subject->isOutputEnabled($moduleName));
+    }
+
+    public function provideIsOutputEnabled(): Generator
+    {
+        yield 'Mage_Core' => [
+            true,
+            'Mage_Core',
+        ];
```

Comment:
```suggestion
        yield 'Mage_Adminhtml' => [
            true,
            null,
        ];
        yield 'Mage_Core' => [
            true,
            'Mage_Core',
        ];
```

Not tested but I think this would work to make sure null param works

Reply:
works, just renamed and added test. thanks.